parser.messages

```diff
@@ -57,9 +57,10 @@ example "(x + 1)", is now expected to construct a variable declaration
 
 program: VAR IDENTIFIER TRIPLE_DOT 
 ##
-## Ends in an error in state: 99.
+## Ends in an error in state: 31.
 ##
-## instruction -> VAR variable . EQUAL expression [ NEWLINE ]
+## var_def -> VAR variable . EQUAL expression [ RBRACKET NEWLINE COMMA ]
+## var_def -> VAR variable . [ RBRACKET NEWLINE COMMA ]
```

Comment:
Shouldn't the error message be updated for the new production rule that was added?